Source/Initialization/WarpXInitData.cpp

```diff
@@ -906,19 +895,6 @@ WarpX::InitLevelData (int lev, Real /*time*/)
     if ((m_p_ext_field_params->E_ext_grid_type == ExternalFieldType::parse_ext_grid_function)
         && (lev <= maxlevel_extEMfield_init)) {
```

Comment:
This part of the code (for `E`) does not seem to be doing the same thing as the above code (for `B`), or did I miss something?

Reply:
The difference is just that the external E-field is used to update `ECTRhofield` (if EB's are ON and the ECT solver is used). But the B-field does not have an equivalent field that needs to be updated.